package.json

```diff
@@ -40,7 +40,8 @@
     ]
   },
   "devDependencies": {
-    "@checkup/cli": "^1.0.0-beta.11",
-    "@checkup/core": "^1.0.0-beta.11"
+    "@checkup/cli": "^1.0.0-beta.13",
+    "@checkup/core": "^1.0.0-beta.13",
+    "upper-case-first": "^2.0.2"
```

Comment:
I'd slightly prefer we use `lodash.startcase` over this package, but not a huge deal.